javascript/ql/src/semmle/javascript/frameworks/Express.qll

```diff
@@ -733,7 +736,7 @@ module Express {
   /**
    * An Express router.
    */
-  class RouterDefinition extends InvokeExpr {
+  class RouterDefinition extends Expr {
```

Comment:
Hm, an unfortunate breaking change. Would it make sense to add a case in `isRouter` instead of this change? That's also how we incorporate type information in here.